docs/markdown/i18n-module.md

```diff
@@ -26,5 +26,6 @@ This merges translations into a text file using `msgfmt`. See [custom_target](ht
 
 * `po_dir`: directory containing translations, relative to current directory
 * `type`: type of file, valid options are `'xml'` (default) and `'desktop'`
+* `args`: list of extra arguments to pass to `msgfmt`
```

Comment:
This should mention that it is only available since 0.41.0 (doc updates go live upon merging)